gramineproject/gramine
# Remove python3-protobuf dependency from packages

## Description of the changes 

See commit message for rationale.

## How to test this PR? 

- check if general CI passed (CI works with oot driver)
- download the packages from pkg-deb-debian11 and install them on dcap system, check as usual
